examples/lessons/10/app.js

```diff
@@ -3,6 +3,11 @@ import {AnimationLoop, Texture2D, loadFile, setParameters} from '@luma.gl/core';
 import {addEvents} from '@luma.gl/addons';
 import {Matrix4, radians} from 'math.gl';
 import {loadWorldGeometry, World} from './world';
+/* eslint-disable complexity */
```

Comment:
Use `// eslint-disable-next-line complexity` in front of each function/method instead of globally disabling it.